commands/timeout.js

```diff
@@ -24,58 +36,52 @@ module.exports = {
       return;
     }
 
-    for (var [id, user] of message.mentions.users) {
```

Comment:
Screw it, it fails with some random error in that lib (because it's not properly error handled) but it doesn't crash the bot.

I'll class this as acceptable. 😬